AdventOfCode/GenerateSolutionFiles.ps1

```diff
@@ -0,0 +1,44 @@
+﻿param(
+    [int]$Year = (Get-Date).Year
+)
+
+$year = 2015
```

Comment:
I'll test the script myself later but this looks to be an unused variable, could probably be removed?

Reply:
Accidental Testing Variable left in. This would force `$year` to be 2015 for all tests, overwriting the users input. Will be fixed in next push.

Reply:
Fixed in 1126f4b